data/remnant/remnant.txt

```diff
@@ -518,10 +648,60 @@ phrase "friendly remnant"
 	word
 		" "
 	word
-		"They're the best!"
+		"They are the best!"
 		"You should definitely try it!"
 		"Definitely better than those outfits in your space."
 
+phrase "remnant outfit recommendations (trusted)"
+	word
+		"Since"
+	word
+		" "
+	word
+		"you have"
+		"the prefects gave you"
+		"our elders allowed you"
+		"our government granted you"
+	word
+		" "
+	word
+		"a license"
+		"a permit"
+		"permission to equip our outfits"
+	word
+		", "
+	word
+		"I"
+		"our experts"
+		"our veterans"
+		"my friends"
+		"we"
+	word
+		" "
+	word
+		"would suggest"
+		"would recommend"
+	word
+		" "
+	word
+		"equipping"
+		"getting"
+		"installing"
+		"outfitting"
+	word
+		" "
```

Comment:
Shouldn't there be an 'a' here?

Reply:
No, not in this case. 

The following piece is 
```
	phrase
		"remnant inhibitor cannon"
		"remnant thrasher cannon"
		"remnant emp torpedo bay"
		"remnant salvage scanner"
```
Which means each of those pieces is actually pulling from farther down in the document (remembering that each "phrase" is actually a generated sentence, not literally the next in parentheses there.)

So for example, the `phrase "remnant inhibitor cannon"` is actually generating something like "some Inhibitor Cannons. Those weapons will disrupt your target's steering and prevent them from escaping!" (there's a lot of options, that's just one of them)

So each of those phrases that can be inserted in there should include their own "a" or alternative.